tests/std/tests/GH_001914_Cached_position/env.lst

```diff
@@ -0,0 +1,4 @@
+# Copyright (c) Microsoft Corporation.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+

```

Comment:
New tests need to be mentioned in `test.lst`, or they won't be run in the internal harness.

Additionally, we conventionally use lowercase for the parts of the test name after the bug database (so this should be `cached_position`).

Reply:
🤦‍♂️